recipe/meta.yaml

```diff
@@ -11,13 +11,18 @@ source:
 
 build:
     number: 0
+    script:
+        - python setup.py build_ext --inplace --compiler=mingw32  # [win and py<35]
+        - python setup.py install --skip-build  # [win and py<35]
+        - python setup.py install --single-version-externally-managed --record record.txt  # [not (win and py<35)]
```

Comment:
Perhaps we need to link the C math library too (possibly inside setup.py for gridfill) since the code uses `fmax` and `fabs` from the C math library. I've no idea really, just guessing blindly!